datahub-web-react/src/app/entity/user/UserInfoSideBar.tsx

```diff
@@ -114,12 +135,23 @@ export default function UserInfoSideBar({ sideBarData, refetch }: Props) {
     };
     const dataHubRoleName = dataHubRoles && dataHubRoles.length > 0 && (dataHubRoles[0]?.entity as DataHubRole).name;
 
+    let countryName;
+    const findCountryName = (code) => {
+        try {
+            countryName = countries[code].name;
+        } catch (error) {
+            countryName = '';
```

Comment:
Instead of doing this, can we first parse country name above, and then below only show if the countryName is not null?

also, i'd move this utility function findCountry name to a utils file - it is not belonging in the component itself. 

const maybeCountryName = getCountryName(countryCode);

....

and inside of getCountryName, you'd simply return NULL if either a) countryCode is null or undefined or b) countryCode cannot successfully be parsed

{ maybeCountryName ? <> : null } as you have today

Reply:
I have updated the PR with these comments, let me know if you have further feedback @jjoyce0510